paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormFragment.kt

```diff
@@ -377,7 +439,6 @@ internal class USBankAccountFormFragment : Fragment() {
     @Composable
     private fun AccountDetailsForm(
         bankName: String?,
-        displayName: String?,
```

Comment:
in some cases it is the bank name, and in other cases it is the institution name.  Maybe displayName is better to capture both cases?